packages/react-ui/src/components/molecules/UploadField/UploadFieldBase.d.ts

```diff
@@ -0,0 +1,22 @@
+import React, { Ref } from 'react';
+import { TextFieldProps } from '@mui/material/TextField';
+import { InputProps } from '@mui/material';
+
+export type UploadFieldBaseProps = Omit<TextFieldProps, 'placeholder' | 'error'> & {
+  name?: string;
+  multiple?: boolean;
+  handleReset?: () => void;
+  handleOpen?: () => void;
+  dragOver?: boolean;
+  error?: string | React.ReactNode;
```

Comment:
This field looks like was passed to TextField and thus previously was boolean.

In MUI textfields there is convention that `error` is "only" a boolean saying whether to render it as red. If text or node is needed it has to land in `helperText`. I think it's good to follow convention and make it back `boolean`

(yes, this is internal class, but it "defined" implementation of `error` handling in `UploadField`)

Reply:
Also not sure what if that is expected when both `error` and `helperText` are provided:

![image](https://github.com/CartoDB/carto-react/assets/1507542/99a0fafb-e5da-4abd-8630-d5ef81d6d484)

Reply:
I get your point, I've reverted the change to keep `error` as boolean, if we need double `helperText` we'll manage per use case.